code/globalincs/globals.h

```diff
@@ -32,8 +32,8 @@
 #define SHORT_CALLSIGN_PIXEL_W	80		// max width of short_callsign[] in pixels
 
 // from ship.h
-#define MAX_SHIPS					400			// max number of ship instances there can be.DTP; bumped from 200 to 400
-#define SHIPS_LIMIT					400			// what MAX_SHIPS will be at release time (for error checking in debug mode); dtp Bumped from 200 to 400
+#define MAX_SHIPS					500		// max number of ship instances there can be.DTP; bumped from 200 to 400, then to 1000 in 2022
+#define SHIPS_LIMIT					500		// what MAX_SHIPS will be at release time (for error checking in debug mode); dtp Bumped from 200 to 400, then to 1000 in 2022
```

Comment:
Comments needs to be updated to 500 instead of 1000 :)

Reply:
details... details...